_includes/context/priority.md

```diff
@@ -1,3 +1,98 @@
+<a class="anchor" id="contexttoolbarpriority"></a><a class="anchor" id="contextformpriority"></a>
+
+### Troubleshooting context toolbar and context form conflicts
+
+There are situations where custom context toolbars or custom context forms may conflict with:
+
+  - Context toolbars or context forms provided by the editor,
+  - Context toolbars or context forms provided by the [Quick Toolbars (`quickbars`) plugin]({{site.baseurl}}/plugins/quickbars/),
+  - Other custom context toolbars or custom context forms.
+
+{% if site.productmajorversion < 6 %}
+How these conflicts are resolved depends on the {{site.productname}} version.
+
+- For {{site.productname}} 5.3 or newer, see: [Determining the display priority of context toolbars and context forms]().
+- For {{site.productname}} 5.0, 5.1, or 5.2; see: [Determining the display priority of context toolbars and context forms (Legacy behavior)]().
+{% endif %}
+
+#### Determining the display priority of context toolbars and context forms
+
+There are three settings that determine the priority of context toolbars and context forms: `scope`, `predicate`, and `position`.
+
+- `scope` - Sets the context toolbar or form as either: specific to certain types of content (`node`), or a general (global) toolbar or form (`editor`).
+- `predicate` - A function that tests in the current context position against one or more conditions, returning a boolean value.
+- `position` - Sets where the context toolbar or form is rendered, relative to the current section (`selection`, `node`, or `line`).
```

Comment:
`section` probably isn't a very clear term here. I'd change this to `selection` or `context` instead.